api/openapi/v1/core-metadata.yaml

```diff
@@ -604,19 +551,17 @@ paths:
           type: string
       responses:
         200:
-          description: boolean indicating success of update
+          description: Boolean indicating success of update
```

Comment:
I don't think this is the same phrasing we're using elsewhere for 200 responses.